packages/containers/src/SelectObject/SelectObject.module.scss

```diff
@@ -37,36 +35,18 @@
 				}
 			}
 
-			margin-top: $padding-smaller;
+			margin-top: 4px;
```

Comment:
No use of coral token here ?

Reply:
I could , but as this is the only part of containers that would require it , i decided to not add a dependency here and let it live (or die) as it is 
But it can be discussed of course